Cython/Compiler/ExprNodes.py

```diff
@@ -5975,6 +5995,20 @@ def compile_time_value(self, denv):
         except Exception as e:
             self.compile_time_value_error(e)
 
+    def calculate_constant_result(self):
+        if self.function.is_attribute and self.function.obj.is_literal:
+            method = self.function.constant_result
+            if inspect.isbuiltin(method):
```

Comment:
Maybe. I've added `ismethod()` as well, although I'd expect other implementations to follow CPython's semantics in cases like this. It could even be enough to just test `callable()`, given that we already know that we're dealing with an attribute of a literal.